packages/styles/scss/patterns/sub-patterns/content-item/_content-item.scss

```diff
@@ -28,6 +28,10 @@
       color: $text-01;
       @include carbon--type-style('body-long-02', true);
     }
+
```

Comment:
One note on the styling is the CTA extends the whole block, so I wonder if we need to add a float, or `inline-block` to contain it a little better.

![image](https://user-images.githubusercontent.com/3793636/74168280-19c79a80-4bef-11ea-9bcd-0eb4ad9a9e8f.png)

Reply:
yeah I think this will need to be addressed in the CTA component

Reply:
<img width="688" alt="Screen Shot 2020-02-11 at 10 26 27 AM" src="https://user-images.githubusercontent.com/54281166/74250692-fcf79980-4cb8-11ea-8d68-997427921dc6.png">

fixed